public/static/locales/en/form-builder.json

```diff
@@ -249,13 +249,6 @@
     "forOtherEnquiriesPart2": "to find the department or agency that can support you.",
     "theGCFormsTeam": "The GC Forms team provides support to people who work in the government. We cannot give general advice to the public about the forms that we host.",
     "backToForms": "Back To Forms"
-  },
```

Comment:
Not sure why this change is cannot be seen in the pull request `Files changed` section but you have somehow deleted strings that are still being used.

Reply:
See https://github.com/cds-snc/platform-forms-client/pull/1342/commits/a267993f1a53ea331d9e4bbfb2487d80f0d14355#diff-54857a04f57e243d2aad32b79912c9e694f5869d7cf69b759ed958da0643ba96L252

Reply:
From what I can see, those strings are still there, though they were strangely moved

Reply:
I'll mark this as resolved since the keys are there but wow very strange